js/app.js

```diff
@@ -48,4 +48,28 @@ btnSubmit.addEventListener('click', (e) => {
     error.classList.remove('.alert');
     error.classList.add('.off-alert');
   }, 100);
-});
\ No newline at end of file
+});
+
+// storage
+
+const information = [];
+const store = (e) => {
+  e.preventDefault();
+  const info = {
+    names: document.getElementById('fullname').value,
+    email: document.getElementById('email').value,
+    message: document.getElementById('message').value,
+  };
+
+  information.push(info);
+  // document.forms[0].reset();
+
+  localStorage.setItem('formData', JSON.stringify(information));
+};
+const feedback = JSON.parse(window.localStorage('formData'));
```

Comment:
- [ ] Great job 👍 . Unfortunately, no data is saved in local storage. There's an error that pops up in the console please fix this 🙏 

![image](https://user-images.githubusercontent.com/74814780/181619821-a14bcf69-448d-4b8b-b65f-af43b782958e.png)